mods/common/languages/en.ftl

```diff
@@ -502,11 +502,16 @@ replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug
 incompatible-replay-title = Incompatible Replay
 incompatible-replay-prompt = Replay metadata could not be read.
 -incompatible-replay-recorded = It was recorded with
-incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version: { $version }.
-incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod: { $mod }.
-incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } an unavailable mod: { $mod }.
-incompatible-replay-incompatible-version = { -incompatible-replay-recorded } an incompatible version: { $version }.
-incompatible-replay-unavailable-map = { -incompatible-replay-recorded } an unavailable map: { $map }.
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } an unknown version:
+    { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } an unknown mod:
+    { $mod }.
```

Comment:
In these cases `$version` and `$mod` are `null` so the game crashes:
```
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object.
   at OpenRA.TranslationExts.ToFluentType(Object value) in /home/ivaylo/code/OpenRA/OpenRA.Game/TranslationExts.cs:line 41
   at OpenRA.Translation.TryGetString(String key, String& value, IDictionary`2 arguments) in /home/ivaylo/code/OpenRA/OpenRA.Game/Translation.cs:line 100
   at OpenRA.Mods.Common.Widgets.ConfirmationDialogs.ButtonPrompt(ModData modData, String title, String text, Dictionary`2 titleArguments, Dictionary`2 textArguments, Action onConfirm, Action onCancel, Action onOther, String confirmText, String cancelText, String otherText) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs:line 43
   at OpenRA.Mods.Common.Widgets.Logic.ReplayUtils.IncompatibleReplayDialog(String text, Dictionary`2 textArguments, ModData modData, Action onCancel) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayUtils.cs:line 78
   at OpenRA.Mods.Common.Widgets.Logic.ReplayUtils.PromptConfirmReplayCompatibility(ReplayMetadata replayMeta, ModData modData, Action onCancel) in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayUtils.cs:line 62
   at OpenRA.Mods.Common.Widgets.Logic.ReplayBrowserLogic.WatchReplay() in /home/ivaylo/code/OpenRA/OpenRA.Mods.Common/Widgets/Logic/ReplayBrowserLogic.cs:line 771
```

We don't need to pass any arguments - the mod/version is simply unknown.